rootfs/etc/dconf/db/local.d/01-dark-theme

```diff
@@ -0,0 +1,3 @@
+[org/gnome/desktop/interface]
```

Comment:
Dark mode is ugly an inconsistent. Let's not turn it on by default.

Reply:
It's inconsistent if not set on all the gym settings. I'm not only setting it here but for GTK 3.0 also. I prefer dark mode for OLED screens but if you think is not good, we can just revert it.

Reply:
Well, for example, I just tried installing gedit via flatpak and it doesn't honor the dark mode setting. I think we leave the default light mode to keep consistency with most apps.

Reply:
Do you have xdg-portal-gnome installed?

Reply:
No, is that needed?

Reply:
I think we would need to set `GTK_THEME_VARIANT=dark` globally for it to work.